app/src/main/java/com/android/wardrobeManager/ui/images/DesignFilterManager.java

```diff
@@ -0,0 +1,38 @@
+package com.android.wardrobeManager.ui.images;
+
+import android.graphics.Bitmap;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DesignFilterManager {
+
+
+    protected Map<String, DesignFilter> filters;
+
+    public static Bitmap filterColorStyle(String colorStyle, Bitmap bitmap, Bitmap ref, int[] colors) {
+        return new DesignFilterManager().filter(colorStyle, bitmap, ref, colors);
+    }
+
+    public DesignFilterManager() {
+        filters = new HashMap<>();
+
+
+        filters.put("primary_secondary", new PrimarySecondary());
+        filters.put("thin_stripes", new ThinHorizontalStripes());
+        filters.put("wide_stripes", new WideHorizontalStripes());
+        filters.put("thin_horizontal_stripes", new ThinHorizontalStripes());
+        filters.put("wide_horizontal_stripes", new WideHorizontalStripes());
+        filters.put("thin_vertical_stripes", new ThinVerticalStripes());
+        filters.put("wide_vertical_stripes", new WideVerticalStripes());
```

Comment:
Good job using the Map here. Make sure the string values in the resources folder reflect these values exactly. That way they will show up in the advanced edit screen